Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -308,22 +321,18 @@ public partial class BlockPolicySource
 
         internal static TxPolicyViolationException ValidateNextBlockTxRaw(
             BlockChain<NCAction> blockChain,
+            IActionTypeLoader actionTypeLoader,
             Transaction<NCAction> transaction,
             ImmutableHashSet<Address> allAuthorizedMiners)
         {
-            // Avoid NRE when genesis block appended
-            // Here, index is the index of a prospective block that transaction
-            // will be included.
-            long index = blockChain.Count > 0 ? blockChain.Tip.Index : 0;
-
-            if (transaction.Actions.Count > 1)
+            if (((ITransaction)transaction).CustomActions?.Count > 1)
             {
                 return new TxPolicyViolationException(
                     $"Transaction {transaction.Id} has too many actions: " +
-                    $"{transaction.Actions.Count}",
+                    $"{((ITransaction)transaction).CustomActions?.Count}",
                     transaction.Id);
             }
-            else if (IsObsolete(transaction, index))
+            else if (IsObsolete(transaction, actionTypeLoader, blockChain.Tip.Header))
```

Comment:
Because of this part, the tests are failing while appending a genesis block.

This problem will be resolved after bumping Libplanet submodule which https://github.com/planetarium/libplanet/pull/2653 is merged.

Reply:
I resolved it in [cad9df9](https://github.com/planetarium/lib9c/pull/1580/commits/cad9df943f6150e8b342df67790afc511d814a7b)